File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
# 문제 url = https://school.programmers.co.kr/learn/courses/30/lessons/12930?language=python3

def solution(str):
  answer = ''
  count = 0
  for i in str:
    if i == ' ':
      answer += ' '
      count = 0
      continue
    answer += i.upper() if count % 2 == 0 else i.lower()
    count += 1
  return answer

File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    var answer = '';
    let j = 0;
    for (let i =0; i<s.length; i++){
        if(s[i] == ' '){
            answer += ' '; j = 0; continue
        }
        answer += j%2 ==0 ? s[i].toUpperCase() : s[i].toLowerCase(); j++;
    }
    return answer;
}